packages/backend/src/apps/google-docs/triggers/new-doc-created/index.js

```diff
@@ -0,0 +1,34 @@
+import defineTrigger from '../../../../helpers/define-trigger.js';
+import newDocuments from './new-doc-created.js';
+
+export default defineTrigger({
+  name: 'New document created',
```

Comment:
```suggestion
  name: 'New document',
```

Let's update the folder name with `new-document` as well.